db/schema.rb

```diff
@@ -401,6 +402,8 @@
     t.boolean "race_refused_to_answer"
     t.boolean "latest_assessment_symptomatic", default: false
     t.date "first_positive_lab_at"
+    t.string "legacy_primary_language"
+    t.string "legacy_secondary_language"
```

Comment:
This looks like an artifact of the 1344 migration. You need to roll that back locally when switch branches.

Reply:
resolved